bnb/getBNBData

```diff
@@ -88,7 +88,7 @@ def main():
             retcode += 1
             continue
         if r.headers['content-type'] != 'application/x-zip-compressed':
-            print(f"Unexpected content type {r.headers['content-type']} for {zipurl}")
+            print(f"Warning: Unexpected content type {r.headers['content-type']} for {zipurl}")
             retcode += 1
```

Comment:
Is this change needed? - The return code is still >0 and therefore I am not sure that it makes sense to change the wording here...

Reply:
This change is for humans. "Unexpected content type" sounds like an error for me, "Warning: Unexpected content type" makes it clear that it's only a warning, but the result can still be valid.

Tagging log messages as info, warning and error is common practice.